Frontend/voitheia-App/src/app/components/user/login/login.component.scss

```diff
@@ -0,0 +1,41 @@
+@import '../../../../variables.scss';
+
+
+#login_header {
+    color: black;
+    font-weight: 700;
+    text-align: center;
+    margin-bottom: 3rem;
+}
+
+
+.mat-input-element {
+    padding: 1.2rem;
+    font-size: 70%;
+    color: black;
+}
+
+
+#login_submit_btn {
+    margin: auto;
+    width: 50%;
+    margin-top: 1rem;
+    margin-bottom: 2rem;
+  
```

Comment:
Should be handled globally to have the same design everywhere

Reply:
moved to style.scss for all occurences